edge-functions/serverside-analytics/pages/api/views.ts

```diff
@@ -0,0 +1,29 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import supabase from '../../lib/supabase'
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const homeQuery = supabase
+      .from('analytics')
+      .select('*', { count: 'exact' })
+      .eq('slug', '/')
+      .then((res) => res.count)
+
+    const otherQuery = supabase
+      .from('analytics')
+      .select('*', { count: 'exact' })
+      .eq('slug', '/other')
+      .then((res) => res.count)
```

Comment:
Why are we using Supabase for analytics, aren't there better tools that are built for that? Unless I'm missing something 🤔